pkg/container/device.go

```diff
@@ -6,13 +6,49 @@ import (
 	"strings"
 
 	devicecfg "github.com/cri-o/cri-o/internal/config/device"
+	"github.com/cri-o/cri-o/server/cri/types"
 	"github.com/cri-o/cri-o/utils"
 	securejoin "github.com/cyphar/filepath-securejoin"
 	"github.com/opencontainers/runc/libcontainer/devices"
 	rspec "github.com/opencontainers/runtime-spec/specs-go"
 	"github.com/pkg/errors"
 )
 
+const (
+	// kubelet/device-plugins can set this annotation to "true" to opt-out for getting Devices
+	// UID/GID set as RunAsUser/RunAsGroup.
+	disableDeviceOwnershipFromSecurityContextAnnotation = "io.kubernetes.cri.DisableDeviceOwnershipFromSecurityContext"
```

Comment:
this should go in  `pkg/annotations`

Reply:
We should add this annotation to the allowed annotations docs, too.

Reply:
OK, will do. Do you have unit tests I should add for this?